timeless/restaurants/models.py

```diff
@@ -94,7 +94,7 @@ class TableReservation(DB.Model):
     reservation = DB.relationship("Reservation", back_populates="tables")
 
 
-class Table(PosterSyncMixin, DB.Model):
+class Table(TimestampsMixin, PosterSyncMixin, DB.Model):
```

Comment:
@Vatavuk I look at this and realize it's useless to add this mixin to every model with other mixins. I think this mixin will be used in 90% models, may be create an abstract Model like this: https://docs.sqlalchemy.org/en/latest/orm/extensions/declarative/mixins.html#augmenting-the-base
WSYT? If you are agree, let's make a puzzle

Reply:
@vladarefiev couldn't agree more